app/model/Campaign.scala

```diff
@@ -18,7 +18,7 @@ case class Campaign(
   startDate: Option[DateTime] = None,
   endDate: DateTime,
   targets: Map[String, Long] = Map.empty,
-  campaignTargets: Option[Map[String, Map[String, Long]]] = None,
+  campaignTargets: Option[Map[String, Map[String, Long]]] = Some(Map.empty),
```

Comment:
If this is never none, should it be optional?

Reply:
Not all campaigns have this field at the moment which causes it to break.